benchmark_runner/jupyterlab/templates/summary_report/summary_report_widgets.py

```diff
@@ -50,6 +51,20 @@ def get_compared_ocp_versions():
             # Handle the case when compared_ocp_versions is not defined
             return None
 
+    def version_key(self, version):
+        """
+        This method splits the version string into parts and returns the numeric parts
+        @param version:
+        @return:
+        """
+        # Split the version string into parts
+        parts = version.replace('-rc.', '-ec.').split('.')
```

Comment:
Why this?  It at least needs a comment.

Reply:
We remove characters from versions like '4.13.0-rc.4' and '4.13.0-ec.4' to enable proper ordering

Reply:
Perfect.  Approving.